### PR TITLE
Replaced MD5 with SHA256 for HashAlgorithm.ComputeHash examples

### PR DESCRIPTION
**Pull request by rpetrusha**
## Replaced MD5 with SHA256 for HashAlgorithm.ComputeHash examples

Related to dotnet/docs#6809

### CODE REVIEW

**Comment by rpetrusha** [comment]
@bartonjs and @morganbr, could you please take a look at this PR and also dotnet/dotnet-api-docs#535, please?

File: snippets/csharp/api/system.security.cryptography/hashalgorithm/example1.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                Console.WriteLine("The hashes are the same.");
+            }
+            else
+            {
+                Console.WriteLine("The hashes are not same.");
+            }
+        }
+    }
+
+    static string GetSHA256Hash(SHA256 sha256Hash, string input)
```
Comment (bartonjs) [inline snippets/csharp/api/system.security.cryptography/hashalgorithm/example1.cs]: I'd expect `GetSHA256Hash` to own creating the hash object.  `GetHash(HashAlgorithm hashAlgorithm, string input)`, on the other hand, makes sense.

File: snippets/csharp/api/system.security.cryptography/hashalgorithm/example1.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            sBuilder.Append(data[i].ToString("x2"));
+        }
+
+        // Return the hexadecimal string.
+        return sBuilder.ToString();
+    }
+
+    // Verify a hash against a string.
+    static bool VerifySHA256Hash(SHA256 sha256Hash, string input, string hash)
```
Comment (bartonjs) [inline snippets/csharp/api/system.security.cryptography/hashalgorithm/example1.cs]: (Similar to the creator, I'd expect `VerifyHash(HashAlgorithm hashAlgorithm, string input, string hash)`)

File: snippets/csharp/api/system.security.cryptography/hashalgorithm/example1.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    {
+        // Hash the input.
+        var hashOfInput = GetSHA256Hash(sha256Hash, input);
+
+        // Create a StringComparer an compare the hashes.
+        var comparer = StringComparer.OrdinalIgnoreCase;
```
Comment (bartonjs) [inline snippets/csharp/api/system.security.cryptography/hashalgorithm/example1.cs]: `var` => `StringComparer`.

File: snippets/visualbasic/VS_Snippets_CLR_Classic/classic SHA256 Example/VB/source.vb [hunk 4]
```diff
@@ -2,68 +2,59 @@
...
+
+        Dim targetDirectory As String = args(0)
+        If Directory.Exists(targetDirectory) Then
             ' Create a DirectoryInfo object representing the specified directory.
-            Dim dir As New DirectoryInfo(directory)
+            Dim dir As New DirectoryInfo(targetDirectory)
             ' Get the FileInfo objects for every file in the directory.
             Dim files As FileInfo() = dir.GetFiles()
             ' Initialize a SHA256 hash object.
             Dim mySHA256 As SHA256 = SHA256Managed.Create()
```
Comment (bartonjs) [inline snippets/visualbasic/VS_Snippets_CLR_Classic/classic SHA256 Example/VB/source.vb]: SHA256Managed => SHA256; and the VB equivalent of a `using` statement (perhaps just manually calling Dispose later?)

**Comment by rpetrusha** [comment]
Thanks for the thorough code review, @bartonjs!

**Pull request merged**